lib/prompts/expand.js

```diff
@@ -222,11 +222,13 @@ Prompt.prototype.validateChoices = function (choices) {
  * @return {String} The rendered choices key string
  */
 Prompt.prototype.generateChoicesString = function (choices, defaultIndex) {
-  var defIndex = 0;
+  var defStr = this.opt.choices.pluck('key');
```

Comment:
I see you reverted the use of `this.opt.choices.length`, any reason for that? Doesn't look like the pluck is useful here as we only want the length.